commercial/app/views/hosted/guardianHostedVideo.scala.html

```diff
@@ -0,0 +1,120 @@
+@import common.commercial.HostedPage
+@(page: HostedPage)(implicit request: RequestHeader)
+
```

Comment:
Could include surrounding page template here (so that the controller only needs to know about one template):
```
@guardianHostedPage(page) {
...
}
```